monkey/infection_monkey/network/windowsserver_fingerprint.py

```diff
@@ -3,14 +3,11 @@
 """
 
 import logging
-import subprocess
 
 import nmb.NetBIOS
 from impacket.dcerpc.v5 import epm, nrpc, transport
 
-import infection_monkey.config
 from infection_monkey.network.HostFinger import HostFinger
-from infection_monkey.utils.environment import is_windows_os
 
```

Comment:
Why is this part of the same commit/PR?

Reply:
Making new PR's for small cleanups like this would be annoying. It's on a separate, descriptive commit though.